services/expiration/expiration-notification.server.ts

```diff
@@ -137,57 +164,48 @@ const updateNotificationStatus = (type: RecordType, id: number) => {
 };
 
 // fetch records by status based on type
-const getRecordsByExpiration = (type: RecordType) => {
-  switch (type) {
-    case RecordType.Certificate:
-      return prisma.certificate.findMany({
-        where: {
-          validTo: {
-            lte: dayjs().add(1, 'M').toDate(),
-          },
-          OR: [
-            {
-              lastNotified: null,
-            },
-            {
-              lastNotified: dayjs()
-                .subtract(NOTIFICATION_FREQUENCY * 4, 'd')
-                .toDate(),
-            },
-          ],
+const getExpiringCertificates = () => {
+  return prisma.certificate.findMany({
+    where: {
+      validTo: {
+        lte: dayjs().add(1, 'M').toDate(),
```

Comment:
Sometimes you use `30 days` and sometimes `1 month`.  Should you do the same everywhere?